### PR TITLE
feat(KHP3-5778): toggle lab order picking based on pending bill status

### PR DESCRIPTION
**Pull request by donaldkibet**
## Requirements

- [x] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [x] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide).
- [x] I checked for feature overlap with [**existing widgets**](https://om.rs/directory).

## Summary

see https://thepalladiumgroup.atlassian.net/browse/KHP3-5778

## Screenshots
https://github.com/palladiumkenya/kenyaemr-esm-3.x/assets/28008754/cfb61c9f-8e17-4e68-b0d9-4c086221dc8e

## Related Issue

*None.*

## Other

*None.*

### CODE REVIEW

File: packages/esm-billing-app/src/billable-services/billiable-item/test-order/test-order-action.resource.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  }>(url, openmrsFetch);
+
+  const isEmergencyPatient =
+    data?.data?.results?.[0]?.queueEntry?.priority?.uuid === config?.concepts?.emergencyPriorityConceptUuid;
+  const isInQueue = data?.data?.results?.length > 0;
+  return { isInQueue, isLoading, error, isEmergencyPatient };
+};
+
+export const usePatientBill = (orderUuid: string) => {
+  const url = `ws/rest/v1/cashier/billLineItem?orderUuid=${orderUuid}&v=full`;
```
Comment (ojwanganto) [inline packages/esm-billing-app/src/billable-services/billiable-item/test-order/test-order-action.resource.tsx:39]: should we make this configurable?

**Review by ojwanganto — APPROVED** [review]
Looks good. I have left a single review for action

File: packages/esm-billing-app/src/billable-services/billiable-item/test-order/test-order-action.resource.tsx (hunk 1 above)
Comment (donaldkibet) [reply packages/esm-billing-app/src/billable-services/billiable-item/test-order/test-order-action.resource.tsx:39]: Am not sure, is there a scenario where we foresee making query about the order being paid out of the cashier module?

Comment (ojwanganto) [reply packages/esm-billing-app/src/billable-services/billiable-item/test-order/test-order-action.resource.tsx:39]: We are also integrating with ERPNext and I guess we'll reuse the same UI. Thoughts?

**Pull request merged**